packages/config/src/layer2s/starknet.ts

```diff
@@ -166,8 +166,8 @@ export const starknet: Layer2 = {
     ],
     transactionApi: {
       type: 'starknet',
-      callsPerMinute: 60 * 5,
-      url: 'https://alpha-mainnet.starknet.io',
+      url: 'https://starknet-mainnet.g.alchemy.com/v2/demo',
```

Comment:
I am not sure whether is a pattern, but in projects that use env variables on heroku we do not provide url and calls, which signals it is using custom RPC url